File: game.cs
using System;

namespace HelloWorld
{
    class Program
    {
        static void Main(string[] args)
        {
            Console.WriteLine("Guess a number between one and a thousand!");

            Random random = new Random();
            int randomNumber = random.Next(1, 1000);
            while (true)
            {
                Console.Write("Guess: ");
                int userInput = Convert.ToInt32(Console.ReadLine());
                if (userInput == randomNumber)
                {
                    break;
                }
                else
                {
                    Console.WriteLine("WRONG!");
                }
            }

            Console.WriteLine("Well Done! You got the number. It was " + Convert.ToString(randomNumber));
        }
    }
}

File: game.py
import time
import random
import math

start_time = time.time()
random_number = random.randint(1, 1000)

print("Guess a number between 1 and 1000 and see how long it takes you")

while True:
    guess = int(input("Guess number: "))
    if guess == random_number:
        break
    else:
        print("Wrong!!")

print("Good job!")
print(f"It took you {math.ceil(time.time() - start_time)}.")
